.github/workflows/unittests-gpu.yml

```diff
@@ -79,13 +79,19 @@ jobs:
           cat jobid.log | xargs -i aws s3api wait object-exists --bucket gluon-nlp-dev --key batch/{}/temp/coverage.xml
           cat jobid.log | xargs -i aws s3 cp s3://gluon-nlp-dev/batch/{}/temp/coverage.xml ./coverage.xml
 
-      - name: Upload coverage to Codecov
-        if: ${{ failure() || success() }}
+      - name: Upload coverage to Codecov(For pull request)
+        if: ${{ (failure() || success()) && (github.event_name == 'pull_request' || github.event_name == 'pull_request_target') }}
```

Comment:
Why check for `(failure() || success())`?

Reply:
Thanks, this check should be removed. We do not need to upload code coverage if the tests failed.